src/components/ZeroState/index.js

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import FDS from 'lib/dictionary/js/styleConstants';
+
+const ZeroState = ({ Icon, label, height, ...props }) => (
+  <article
+    className="alignChild--center--center pointer-events-none"
+    style={{ minHeight: height }}
```

Comment:
should `height` have a default of `100%` to fill the parent container by default?

Reply:
I'm hesitant to do that unless you're 100% confident that there won't be regressions. I think it's safer for these cbi-site storybook --> FDS migrations to be mostly identical in display / functionality as it sort of muddies the waters and makes a simple import switch carry a bit more risk. I'd rather introduce those sort of changes either before migration on cbi-site, or post a more vanilla migration.

That all being said the change itself makes sense.